DNN Platform/DotNetNuke.Web/Api/Internal/DnnActionFilterProvider.cs

```diff
@@ -3,17 +3,25 @@
 // See the LICENSE file in the project root for more information
 
 namespace DotNetNuke.Web.Api.Internal
-{
+{
+    using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Web.Http;
     using System.Web.Http.Controllers;
     using System.Web.Http.Filters;
 
-    using DotNetNuke.Common;
+    using DotNetNuke.Common;
+    using DotNetNuke.DependencyInjection.Extensions;
 
     internal class DnnActionFilterProvider : IFilterProvider
-    {
+    {
+        private IServiceProvider container;
+        public DnnActionFilterProvider(IServiceProvider container)

```

Comment:
Is DI going to resolve the global container or a scoped container here?  Do we need to think about scoping?  I would hope it would automatically pull the scoped container, which should already be in use, but want to make sure we don't make the wrong assumption here.

Reply:
As far as I understand it will resolve the scoped provider. @mitchelsellers do you know more about this?

This is the technically correct way to access the `IServiceProvider` instead of using the `Globals.DependencyProvider`. I thought since we can register the `DnnActionFilterProvider` let's just resolve the `IServiceProvider` in the constructor

Reply:
How do we verify this? Do we have an easy way where we can just attach a debugger and check the `IServiceProvider`?